DQM/HLTEvF/plugins/TrigObjTnPHistColl.cc

```diff
@@ -0,0 +1,399 @@
+#include "TrigObjTnPHistColl.h"
+
+#include "FWCore/Common/interface/TriggerNames.h"
+
+namespace{
+  std::vector<float> convertToFloat(const std::vector<double>& vecD){
```

Comment:
More simply
```
return std::vector<float>(vecD.begin(), vecD.end());
```
which should preallocate the right number of elements and avoid resizing.

Reply:
urgh, I feel dumb, how did I not realise I could do this... thanks